dpnp/backend/extensions/blas/blas_py.cpp

```diff
@@ -39,9 +39,9 @@ namespace py = pybind11;
 // populate dispatch tables
 void init_dispatch_tables(void)
```

Comment:
Need either to rename the function or to separate initializing of vectors to `init_dispatch_vectors`, because now it is a mix of inits vectors and tables but with the old function name.

Reply:
The comment isn't actual anymore.